pytorch_blade/cmake_build.py

```diff
@@ -17,98 +17,17 @@
 import shutil
 import re
 from distutils.spawn import find_executable
-import release_version
+from torch_blade_build import TorchBladeBuild, get_fullpath_or_create, check_env_flag
```

Comment:
what's the relationship between cmake_build.py and bazel_build.py
Will they both work for a while?

Reply:
Yes, we currently support CMake and Bazel both.